rust/oak_log/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "oak_log"
+version = "0.1.0"
+authors = ["David Drysdale <drysdale@google.com>"]
+edition = "2018"
+
+[dependencies]
+oak = { path = "../oak" }
+log = { version = "^0.4.5", features = ["std"] }
```

Comment:
I would relax this to `version = "0.4"` too.
Also it seems there is a missing EOL.